### PR TITLE
acquireToken() add id token

### PR DESCRIPTION
**Pull request by kaaaaai**

### CODE REVIEW

File: ios/Classes/MsalAuthPlugin.swift [hunk 1]
```diff
@@ -236,6 +238,10 @@ extension MsalAuthPlugin {
         })
     }
     
+    private func getIDToken(result: @escaping FlutterResult) {
+
```
Comment (nayanAubie) [inline ios/Classes/MsalAuthPlugin.swift:242]: This method is not required more I guess.

Comment (kaaaaai) [reply ios/Classes/MsalAuthPlugin.swift:242]: In my case, the application needs to log in to get an ID token, which is then sent to our own service to get the user information via the ID token, the application does not retain the third login state, and the access token is provided by the service after the service gets the correct user information.

Comment (nayanAubie) [reply ios/Classes/MsalAuthPlugin.swift:242]: Yes I understand, but my point is your function `getIDToken` contains an empty body. so this function is not required in the code

File: example/pubspec.lock
Comment (nayanAubie) [inline example/pubspec.lock:1]: You can upgrade the versions. I can not take it with downgrade

File: lib/models/msal_user.dart [hunk 2]
```diff
@@ -15,12 +15,16 @@ class MsalUser {
   /// Token expiration time in "MillisecondsSinceEpoch".
   final int tokenExpiresOn;
 
+  /// The raw id token if it's returned by the service or nil if no id token is returned.
```
Comment (nayanAubie) [inline lib/models/msal_user.dart:18]: Your comment has the wrong information. how `final String idToken` can be `nil`? it will have a value or blank.

File: ios/Classes/MsalAuthPlugin.swift (hunk 1 above)
Comment (kaaaaai) [reply ios/Classes/MsalAuthPlugin.swift]: Sorry, I misunderstood and have removed unnecessary methods.

File: example/pubspec.lock
Comment (kaaaaai) [reply example/pubspec.lock:1]: ok

File: lib/models/msal_user.dart (hunk 2 above)
Comment (kaaaaai) [reply lib/models/msal_user.dart]: already update it.

File: example/pubspec.lock
Comment (nayanAubie) [reply example/pubspec.lock:1]: Is there any chance you can update the code?

File: ios/Classes/MsalAuthPlugin.swift [hunk 3]
```diff
@@ -235,7 +236,7 @@ extension MsalAuthPlugin {
             result(true)
         })
     }
-    
+
```
Comment (nayanAubie) [inline ios/Classes/MsalAuthPlugin.swift:239]: Please rollback this unnecessary change.

File: ios/Classes/MsalAuthPlugin.swift [hunk 4]
```diff
@@ -401,4 +402,4 @@ extension WKWebView {
     func refreshCookies() {
         self.configuration.processPool = WKProcessPool()
     }
-}
\ No newline at end of file
+}
```
Comment (nayanAubie) [inline ios/Classes/MsalAuthPlugin.swift:405]: Please rollback this unnecessary change.

File: ios/Classes/MsalAuthPlugin.swift (hunk 3 above)
Comment (kaaaaai) [reply ios/Classes/MsalAuthPlugin.swift]: Check the new code now?

File: example/pubspec.lock
Comment (nayanAubie) [reply example/pubspec.lock:1]: You can revert this change and then I am happy to merge

Comment (kaaaaai) [reply example/pubspec.lock:1]: ok, i've revert it

**Review by nayanAubie — APPROVED** [review]
